### ISSUE TITLE
Cannot read property 'oldlines' of undefined

### ISSUE THREAD

**Issue by nwtn** [kpdecker/jsdiff#49]
I'm getting this error when using jsdiff via [grunt-patch](https://github.com/nettantra/grunt-patch). I've posted the issue with details on the grunt-patch repo (https://github.com/nettantra/grunt-patch/issues/3), but the error appears to be coming from jsdiff, so I thought I'd post it here as well. Any ideas what may be causing this issue?

**Comment by kpdecker** [comment]
I think the best thing would be to try to setup a jsfiddle or similar demonstrating the issue, that or open a PR with a failing test case. Issues with repo cases are always fixed faster :)

**Comment by sebastian-zarzycki** [comment]
I have the same problem.

**Comment by kpdecker** [comment]
If an example showing how to reproduce this isn't posted then it will never be fixed.

**Comment by sebastian-zarzycki** [comment]
It chokes on pretty much everything unified. I believe it's because it expects index header or particular number of lines. My diff file started with this and it couldn't parse it
```
--- angular-translate-loader-static-files.js
+++ angular-translate-loader-static-files.js
@@ -42,7 +42,7 @@
```

When removed first two lines:
```
@@ -42,7 +42,7 @@
```

it worked.

When added two lines:

```
Index: angular-translate-loader-static-files.js
===================================================================
--- angular-translate-loader-static-files.js
+++ angular-translate-loader-static-files.js
@@ -42,7 +42,7 @@
```

it worked.

**Comment by kpdecker** [comment]
Released in 1.3.2
